datalad/distribution/update.py

```diff
@@ -90,10 +90,18 @@ def __call__(name=None, dataset=None,
 
         repos_to_update = [ds.repo]
         if recursive:
-            repos_to_update += [GitRepo(opj(ds.path, sub_path))
+            repos_to_update += [GitRepo(opj(ds.path, sub_path), create=False)
```

Comment:
so there were no test for recursive action I guess since this one remains not covered